### PR TITLE
Did you want some more mappings here you go

### PR DESCRIPTION
**Pull request by pyrox0**
Whoops I slipped and dropped these sorry you should have them

Major highlights are dispenser behavior fixes, mapping every unmapped AI Task(no conflicts with #569, I already checked), datafixer stuff, and a bunch of screen mappings.

### CODE REVIEW

**Comment by eristhea** [comment]
DOES MAPPING "AT LEAST THE CLASS NAMES" MEAN THAT ENTITY WILL LOSE IT'S 100% STATUS ONCE THIS IS MERGED?

**Comment by eristhea** [comment]
you're so done for this one

File: mappings/net/minecraft/entity/ai/brain/task/BreezeLongJumpTask.mapping
Comment (eristhea) [inline mappings/net/minecraft/entity/ai/brain/task/BreezeLongJumpTask.mapping]:
```suggestion
	METHOD m_tqdufxwm isTargetOutOfRange (Lnet/minecraft/unmapped/C_nqresjkz;Lnet/minecraft/unmapped/C_usxaxydn;)Z
```
convention!

File: mappings/net/minecraft/entity/ai/brain/task/BreezeLongJumpTask.mapping
Comment (eristhea) [inline mappings/net/minecraft/entity/ai/brain/task/BreezeLongJumpTask.mapping]:
```suggestion
	METHOD m_nljldkhq isFinishedInhaling (Lnet/minecraft/unmapped/C_nqresjkz;)Z
```

File: mappings/net/minecraft/entity/ai/brain/task/BreezeLongJumpTask.mapping
Comment (eristhea) [inline mappings/net/minecraft/entity/ai/brain/task/BreezeLongJumpTask.mapping]:
```suggestion
	METHOD m_cuwdufka isFinishedLongJumping (Lnet/minecraft/unmapped/C_nqresjkz;)Z
```

File: mappings/net/minecraft/entity/ai/brain/task/BreezeLongJumpTask.mapping
Comment (eristhea) [inline mappings/net/minecraft/entity/ai/brain/task/BreezeLongJumpTask.mapping]:
```suggestion
	METHOD m_ggkaclng isTargetTooClose (Lnet/minecraft/unmapped/C_nqresjkz;Lnet/minecraft/unmapped/C_usxaxydn;)Z
```

File: buildSrc/src/main/resources/minecraft_specific_words.txt
Comment (eristhea) [inline buildSrc/src/main/resources/minecraft_specific_words.txt:194]: no thanks, this should be 2 separate words unless I'm missing something

**Review by eristhea — COMMENTED** [review]
looks good!

**Comment by pyrox0** [comment]
> DOES MAPPING "AT LEAST THE CLASS NAMES" MEAN THAT ENTITY WILL LOSE IT'S 100% STATUS ONCE THIS IS MERGED?

NO I FULLY MAPPED ALL OF THEM >:(

File: buildSrc/src/main/resources/minecraft_specific_words.txt
Comment (pyrox0) [reply buildSrc/src/main/resources/minecraft_specific_words.txt:194]: This is needed for some static fields in `net/minecraft/client/gui/screen/Screen`.

Comment (supersaiyansubtlety) [reply buildSrc/src/main/resources/minecraft_specific_words.txt:194]: `Identifier INWORLD_MENU_BACKGROUND = new Identifier("textures/gui/inworld_menu_background.png");`
Maybe it's ok if this differs from the string?

**Review by supersaiyansubtlety — APPROVED** [review]

File: buildSrc/src/main/resources/minecraft_specific_words.txt
Comment (pyrox0) [reply buildSrc/src/main/resources/minecraft_specific_words.txt:194]: What would you suggest it be changed to? I kept it the same as Mojang's mapping, and Yarn's just adds `_TEXTURE` as a suffix.

Comment (supersaiyansubtlety) [reply buildSrc/src/main/resources/minecraft_specific_words.txt:194]: `IN_WORLD_MENU_BACKGROUND`
But I'd like another opinion before deviating from a string.

Comment (eristhea) [reply buildSrc/src/main/resources/minecraft_specific_words.txt:194]: I should have recognised it was a string, I'd rather keep it as inworld instead of deviating

**Review by eristhea — APPROVED** [review]

**Pull request merged**
